modules/app/app-contentstudio/src/main/js/app/issue/view/IssueDetailsDialog.ts

```diff
@@ -396,7 +396,8 @@ export class IssueDetailsDialog extends SchedulableDialog {
                 const countToPublish = this.countTotal();
                 this.updateButtonCount(i18n('action.publish'), countToPublish);
 
-                this.toggleAction(countToPublish > 0 && !depResult.isContainsInvalid());
+                this.toggleAction(countToPublish > 0);
+                this.getButtonRow().getActionMenu().setEnabled(!depResult.isContainsInvalid() && depResult.isAnyPublishable());
 
```

Comment:
Shouldn't it be ``isAllPublishable`` in this condition? The button should be disabled unless **all** of the items are publishable. 

It also doesn't seem to work. Here you can see that the "Publish" action are disabled for selected item:

![image](https://user-images.githubusercontent.com/11612490/28023560-83729236-658e-11e7-9b5e-1cfbd81ec827.png)

But here the button is enabled for the same item (and I think it's because of ``isAnyPublishable``):
![image](https://user-images.githubusercontent.com/11612490/28023647-bf302ee6-658e-11e7-872b-edffb7658aeb.png)